src/gui/main.cpp

```diff
@@ -41,8 +42,8 @@ void warnSystray()
 
 int main(int argc, char **argv)
 {
-    Q_INIT_RESOURCE(client);
-
+    // load the resources
+    const OCC::ResourcesLoader resource;
```

Comment:
This variable is called `resources` in the test utils. You might want to keep the naming consistent.